commands/log.go

```diff
@@ -32,3 +46,69 @@ Outputs event log messages (not other log messages) as they are generated.
 		return re.Emit(r)
 	},
 }
+
+var logLevelCmd = &cmds.Command{
+	Helptext: cmdkit.HelpText{
+		Tagline: "Change the logging level.",
+		ShortDescription: `
+Change the verbosity of one or all subsystems log output. This does not affect
+the event log.
+`,
+	},
+
+	Arguments: []cmdkit.Argument{
+		// TODO use a different keyword for 'all' because all can theoretically
+		// clash with a subsystem name
+		cmdkit.StringArg("subsystem", true, false, fmt.Sprintf("The subsystem logging identifier. Use '%s' for all subsystems.", logAllKeyword)),
+		cmdkit.StringArg("level", true, false, `The log level, with 'debug' the most verbose and 'critical' the least verbose.
+			One of: debug, info, warning, error, critical.
+		`),
+	},
+
+	Run: func(req *cmds.Request, res cmds.ResponseEmitter, env cmds.Environment) error {
+		args := req.Arguments
+		subsystem, level := args[0], args[1]
+
+		if subsystem == logAllKeyword {
+			subsystem = "*"
```

Comment:
if per comment at top of file args 0 is an array of all files then how does this assignment work?

Reply:
yup, this is gone if we make subsystem an option